pages/SpinButton.js

```diff
@@ -0,0 +1,72 @@
+import { useState } from "react";
+
+import Link from "next/link";
+
+function SpinButton() {
+  const [count, setCount] = useState(1);
+
+  const handleChangeCount = (event) => {
+    const input = event.target.valueAsNumber;
+
+    if (input < 1 || input > 3) {
+      return;
+    }
+
+    setCount(event.target.valueAsNumber);
+  };
+
+  const handleMinusCount = () => {
+    if (count <= 1) {
+      alert(`승객 수는 ${1}명 이상이어야 합니다.`);
```

Comment:
```suggestion
      alert('승객 수는 1명 이상이어야 합니다.');
```

왜 굳이 `${1}`이라고 쓰셨을까요 👀 
상수화를 하려다가 까먹으신 걸까요

Reply:
헉 예리킹 ㄷㄷ 상수화하려다가 까먹었습니다.... 다시 돌려놓도록 할게요!! 👍👍👍